iOS/Core/Pixel.swift

```diff
@@ -288,7 +289,7 @@ public class Pixel {
         if isDebugBuild {
             newParams[PixelParameters.test] = PixelValues.test
         }
-        if isInternalUser {
+        if isInternalUser && includedParameters.contains(.isInternalUser) {
```

Comment:
Since there were no active examples of us using this currently, I've not updated any existing pixels to keep using it.